pkg/visualizer/plugin.go

```diff
@@ -106,6 +106,46 @@ func (p Plugin) Translate(dag *core.ResourceGraph) ([]core.File, error) {
 	}, nil
 }
 
+// Translate implements compiler.IaCPlugin - although it's not strictly an IaC plugin, it uses the same API
+func (p Plugin) Generate(dag *core.ResourceGraph, filenamePrefix string) ([]core.File, error) {
+	api := visApi{client: p.Client}
+
+	if validateTypes {
+		types := TypesChecker{DAG: dag}
+
+		if resp, err := api.request(http.MethodGet, `validate-types`, `application/text`, ``, types); err != nil {
```

Comment:
We might want to disable this warning since the new visualizer doesn't have this capability just yet.